portfolio/src/main/java/com/google/sps/servlets/OfficeGame.java

```diff
@@ -72,20 +91,30 @@ public String getAnswer() {
   public void submitAnswer(String guess) {
     lastSubmission = values.get(currentPrompt).equals(guess);
     if(lastSubmission){
-        score += 1;
+      score += 1;
+      if(score > highScore){
+        Entity scoreEntity = new Entity("Score");
+        scoreEntity.setProperty("score", score);
+        datastore.put(scoreEntity);
+        highScore = score;
+      }
     } else {
-        score = 0;
+      score = 0;
     }
   }
 
   public boolean getWinStatus() {
     return lastSubmission;
   }
 
-  public int getScore() {
+  public long getScore() {
```

Comment:
why did you change the type to long?

Reply:
for some reason when you insert an integer into datastore it stores it and returns it as a long